src/main/kotlin/com/coder/gateway/views/steps/CoderWorkspacesStepView.kt

```diff
@@ -659,7 +659,7 @@ class CoderWorkspacesStepView :
             cs.launch(ModalityState.current().asContextElement()) {
                 while (isActive) {
                     loadWorkspaces()
-                    delay(5000)
+                    delay(1000)
```

Comment:
I made this poll more frequently because so it picks up the "Starting" state faster while the CLI is starting the workspace. I'm not sure if there's a good way to switch to fast polling while the CLI is running and then switch back - feel free to make suggestions or commit to the branch!